README.md

```diff
@@ -19,6 +27,16 @@ npm test
 ```bash
 zip -r cl-synth-index-cc.zip .
 ```
+## Run with Docker
```

Comment:
Seems like we're missing a blank newline here.

Reply:
We're still missing the newline here. LGTM otherwise.

Reply:
I really need glasses. Looked at it before but could not see the diff :)

Reply:
No worries :)